examples/test/AttributeFilter_test.js

```diff
@@ -0,0 +1,39 @@
+import { Selector } from 'testcafe';
+import { config } from './utils/config';
+import { loginUsingGreyPages } from './utils/helpers';
+
+fixture('Attribute filter components')
+    .beforeEach(loginUsingGreyPages(`${config.hostname}/attribute-filter-components`))
+    .page(`${config.hostname}/attribute-filter-components`);
+
+test('Dropdown filter', async (t) => {
```

Comment:
Could you update the title of the test so it's saying clearly what is it testing?